internal/v1/handler_blueprints.go

```diff
@@ -47,6 +53,17 @@ func (u *User) CryptPassword() error {
 	return nil
 }
 
+func (u *User) RedactPassword() {
+	redactedPassword := "<REDACTED>"
+	if u.Password != nil {
+		*u.Password = redactedPassword
+	}
+}
+
+func (u *User) IsRedacted() bool {
+	return u.Password == nil || *u.Password == "<REDACTED>"
+}
```

Comment:
_Little bit of copying is worth it when avoiding a new dependency_ -- Rob Pike

👍🏻